packages/neuron-ui/src/locales/en.json

```diff
@@ -210,18 +209,6 @@
     "footer": {
       "fail-to-fetch-tip-block-number": "Cannot fetch tip block number"
     },
-    "menuitem": {
```

Comment:
@Keith-CY I guess we don't use these any more. Can you check and confirm?

Reply:
Yes, they are not used any more.

Reply:
Nice. Thanks!